README.md

```diff
@@ -170,3 +126,82 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+# Deployment to Fly.io
+
+## Install FlyCtl
+
+To work with the Fly platform, you first need to install Flyctl, a command line interface that allows you to do everything from creating an account to deploy to Fly.
+  - Linux:
+    ```  
+    curl  -L https://fly.io/install.sh | sh
+    ```
+  - OSX:
+    ```
+    brew install flyctl
+    ```
+  - Windows:
+    ```
+    iwr https://fly.io/install.ps1 -useb
+    ```  
+    Here is the [Installation Guilde](https://fly.io/docs/hands-on/install-flyctl/)
+
+
+Next authenticate with your fly.io account:
+  ```
+  fly auth login
+  ```
+
+- To make sure everything is working well:    
+  ```
+  fly apps list
+  ```
+The output of the above command will be empty table since you have no apps launched yet
+
+
+## Configure the Project
+
+### Environment Variables
+
+We shouldn't store secrets in source code, so utilizing environmental variables is needed.
+- Run the following from the root of your project
+  ```
+  python3 set_env.py
+  ```
+
+### Deploy App
+
+In this step the app is going to be launched to fly.io.
+- Create and configure the app  
+  ```
+  python fly_manager.py launch <app-name>
+  
+  ```
+    
+This command will create you an app on Fly.io , spin up a postgres instance, and create an app configuration named fly.toml in your project root. fly.toml file contains all app details.
+  
+Copy the DATABASE_URL from the termial output of the above process(fly launch)
+and Update DATABASE_URL in .env file.
+
+- To make sure the app is created successfully:
+  ```
+  fly apps list
+  ```
+  This command prints 3 apps: 
+  1. your app
+  2. database instance and 
+  3. Fly builders: to build docker images
+
+
+#### Deploy
+
+- To deploy the app to the FLY platform.
+  ```
+  python fly_manager.py deploy <app-name>
```

Comment:
Again. I'm sick of typing my app name everywhere. I want to give my username ONCE during setup and then it puts my app name everywhere

Reply:
hmm ok

Reply:
app-name is required for fly lauch only now

Reply:
for deployn it's optional , it can be specified if user has more than one app launched